file-formats/scp1/type/zif_aff_scp1_v1.intf.abap

```diff
@@ -262,9 +244,12 @@ INTERFACE zif_aff_scp1_v1
     "! <p class="shorttext">Business Configuration Set</p>
     "! Metadata information of BC Set
     BEGIN OF ty_main,
+      "! <p class="shorttext">Format Version</p>
+      "! Format version
```

Comment:
You dont have to put title and description here. 
If you dont put it here it will be automatically read from the annotation in zif_aff_types_v1=>ty_format_version and will be written into the schema of SCP1.

Reply:
Removed